_posts/2020-10-06-solidity-dynamic-array-cleanup-bug.md

```diff
@@ -0,0 +1,63 @@
+---
+layout: post
+published: true
+title:  "Solidity Dynamic Array Cleanup Bug"
+date:   2020-10-06
+author: Solidity Team
+category: Security Alerts
+---
+
+On September 17, 2020, a bug in the Solidity code generator was found. The bug is fixed with version 0.7.3
+released on October 6, 2020. The bug is present in all prior versions of Solidity.
+
+We assigned the bug, a severity level of *medium*.
+
+## Dynamic Array Cleanup Bug
+**Summary**: When assigning a dynamically-sized array with types of size at most 16 bytes in storage
+causing the assigned array to shrink, some parts of deleted slots were not zeroed out.
```

Comment:
```suggestion
causing the assigned array to shrink, some parts of the deleted slots were not zeroed out.
```

This sentence is quite hard to read, can we simplify it?

Reply:
Would something like "For a dynamically-sized storage-array, with types of size at most 16 bytes, assignments that require deleting slots did not zero out the deleted slots properly" be better?